CHANGELOG.txt

```diff
@@ -15,6 +15,8 @@
     # Add a new securised view to download dynamic files (ie: uploaded files, files related to entity, files generated by export...).
     # An exception handler for errors 400 (Bad Request) has been added.
     # Apps :
+        * Creme_core :
```

Comment:
The "Apps" section is for all apps, excepted "creme_core" (eg: "Add a new securised view to download dynamic files ..." is about creme_core).

Reply:
Ok ^^